lib/bindings/http/http_receiver.js

```diff
@@ -0,0 +1,61 @@
+const V03Binary = require("./receiver_binary_0_3");
+const V03Structured = require("./receiver_structured_0_3.js");
+const V1Binary = require("./receiver_binary_1.js");
+const V1Structured = require("./receiver_structured_1.js");
+
+class HTTPReceiver {
+  constructor() {
+    this.receivers = {
+      v1: {
+        structured: new V1Structured(),
+        binary: new V1Binary()
+      },
+      v03: {
+        structured: new V03Structured(),
+        binary: new V03Binary()
+      }
+    };
+  }
+
+  accept(headers, body) {
+    const mode = getMode(headers);
+    const version = getVersion(mode, headers, body);
+    switch (version) {
+      case "1.0":
```

Comment:
Perhaps the constant can be used and for "3.0" below:
https://github.com/cloudevents/sdk-javascript/blob/e329d9ac00d6a2e8a29d0aca6d18d7c4eddaa02e/lib/bindings/http/constants.js#L7